PE/__init__.py

```diff
@@ -632,10 +644,15 @@ def parseResources(self):
                     subdata = self.readStructAtRva( dresc.VirtualAddress + dirent.OffsetToData, 'pe.IMAGE_RESOURCE_DATA_ENTRY')
                     # RP BUG FIX - sanity check the subdata
                     if subdata and self.checkRva(subdata.OffsetToData, size=subdata.Size):
-                        langid = name_id & 0x3ff
-                        sublangid = name_id >> 10
-                        langinfo = (subdata.CodePage, langid, sublangid )
-                        rsdirobj.addRsrcData(subdata.OffsetToData, subdata.Size, langinfo )
+                        # sometimes people are bad and they lie to us
+                        try:
+                            langid = name_id & 0x3ff
+                            sublangid = name_id >> 10
+                        except:
```

Comment:
i suspect this is because langid or sublangid are None?  but should we be specific here for the exception?

Reply:
It's actually because name_id ends up being a unicode string because pieces of the header aren't quite right, but we can still possibly recover bits of information.

It's non-essential and I can remove it if need be. It was a nice to have that was a quick fix.